frontend/playground-frontend/src/components/Dashboard/Login.js

```diff
@@ -122,6 +132,16 @@ const Login = () => {
           {isRegistering ? "Log in" : "Register"}
         </a>
       </div>
+
+      {isRegistering && (
+        <div className="reCaptcha">
+          <ReCAPTCHA
+            sitekey={process.env.REACT_APP_CAPTCHA_SITE_KEY}
```

Comment:
explain? @farisdurrani @AmyCGT

Reply:
Not too sure what you mean, but if this is regarding the 'isRegistering' part, then I put that there so that the reCaptcha wouldn't show up if the page were to change to the login screen rather than the sign up screen